src/views/MaintenancePage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <main class="maintenance-page">
+        <div class="maintenance-page_content level">
+            <img
+                class="maintenance-page_img"
+                src="../assets/img/mantenimiento-image.svg"
+                alt="Maintenance Image"
+                title="Maintenance-Image"
+            />
+            <div class="maintenance-page_description level">
+                <div class="maintenance-page_title">
+                    {{ $t('views.maintenance.title') }}
+                </div>
+                <div class="maintenance-page_subtitle">
+                    {{ $t('views.maintenance.subtitle') }}
+                </div>
+            </div>
+            <button-translation></button-translation>
+        </div>
+    </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
```

Comment:
@gonzalo-plaza please add a white line to separe the imports, cause theese are diferents imports, on the one hand, we import a library and on the other hand we are importing the components and also add withe line space, to separe the imports component that '@components'